howto/troubleshoot/landing.md

```diff
@@ -1,147 +1,22 @@
-This section covers some of the most commonly encountered problems and gives instructions for resolving them.
+This topics in this guide describes some commonly encountered problems with Anbox Cloud and provides instructions for resolving them. If you encounter an issue with Anbox Cloud, check if any of the following scenarios help in resolving your issue.
```

Comment:
hmm, not a native speaker. but to me we should use the `describe` and `provide` here because the `topics` are plural